### PR TITLE
yaegi 0.0.4 (new formula)

### PR DESCRIPTION
**Pull request by arbourd**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/yaegi.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+class Yaegi < Formula
+  desc "Yet another elegant Go interpreter"
+  homepage "https://github.com/containous/yaegi"
+  url "https://github.com/containous/yaegi/archive/v0.0.3.tar.gz"
```
Comment (chenrui333) [inline Formula/yaegi.rb]: might be better using this style:
```
  url "https://github.com/containous/yaegi.git",
    :tag      => "v0.0.3",
    :revision => "24db786bb649f292804184900d5e1818e51cd9cb"
```

**Review by chenrui333 — APPROVED** [review]
👍

File: Formula/yaegi.rb (hunk 1 above)
Comment (arbourd) [reply Formula/yaegi.rb]: cool. is it preferred to use git and tags over tarballs now?

Comment (chenrui333) [reply Formula/yaegi.rb]: Although, right now the golang formula are mixed style, but we definitely talk about the git and tags over tarballs, could not find it now. Yeah, the new formula I created are all with tag and revision, they are equivalent to tarball and sha256 though.

cc @fxcoudert

Comment (crunchtime-ali) [reply Formula/yaegi.rb]: Just a smol update regarding yaegi v0.0.4:

```
  url "https://github.com/containous/yaegi.git",
    :tag      => "v0.0.4",
    :revision => "373e3204b4979b8f7545bcde3e440e0fc3c54743"
```

Comment (SMillerDev) [reply Formula/yaegi.rb]: Actually tarbals are prefered cause those don't need an entire git checkout and cache better/faster

**Comment by arbourd** [comment]
@BrewTestBot test this plz 🙏 	

Previous test failed due to CI being down.

**Comment by arbourd** [comment]
@BrewTestBot test this please

previous commit failed due to CI issues.

**Comment by arbourd** [comment]
@BrewTestBot test this please

Cloning is failing, again. 
> Error fetching remote repo 'origin'

**Comment by arbourd** [comment]
anything still blocking this?

**Comment by SMillerDev** [comment]
This version is marked as an alpha and therefore not well suited for homebrew. All submitted software should be marked as stable by upstream.

**Comment by arbourd** [comment]
okay ill redo this when they become "stable".

**Pull request closed without merging**